Testing/gdcmanonMapping/testDeterministicObfuscation

```diff
@@ -0,0 +1,87 @@
+#!/bin/bash
```

Comment:
I did not want to pull a bunch of dependencies into the gdcm project. I took a look at doing it in C++ but gdcm does not really provide any testing facilities, and since I needed to test the command line programs anyways bash seemed reasonable. I thought it ended up pretty straightforward but I agree is not ideal.